app/jobs/archived_intake_email_verification_code_job.rb

```diff
@@ -0,0 +1,11 @@
+class ArchivedIntakeEmailVerificationCodeJob < ApplicationJob
+  retry_on Mailgun::CommunicationError
+
+  def priority
+    PRIORITY_HIGH - 1 # Subtracting one to push to the top of the queue
```

Comment:
is this normal? seems weird to me

Reply:
this seemed weird to me too but it seems like we use this pattern 🙈 maybe there should be `PRIORITY_IMMEDIATE` or soemthing